### PR TITLE
doc/rbd: add namespace information for mirror commands

### PR DESCRIPTION
**Pull request by nbalacha**
Clarifies that some mirror commands hold for either pools or namespaces.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/man/8/rbd.rst [hunk 1]
```diff
@@ -523,26 +523,26 @@ Commands
...
+  Every mirror-enabled image in the pool or namespace will be demoted.
 
 :command:`mirror pool disable` [*pool-name*]
-  Disable RBD mirroring by default within a pool. When mirroring
-  is disabled on a pool in this way, mirroring will also be
-  disabled on any images (within the pool) for which mirroring
+  Disable RBD mirroring by default within a pool or namespace. When mirroring
+  is disabled on a pool or namespace in this way, mirroring will also be
+  disabled on any images (within the pool or namespace) for which mirroring
   was enabled explicitly.
```
[the remark this answers is not in the document]
Comment (anthonyeleven) [reply doc/man/8/rbd.rst:533]: I would probably leave out "whether by default or explicily" but it doesn't hurt to leave it in.

**Review by anthonyeleven — COMMENTED** [review]
I would probably leave out "whether by default or explicily" but it doesn't hurt to leave it in.

File: doc/man/8/rbd.rst (hunk 1 above)
[the remark this answers is not in the document]
Comment (idryomov) [reply doc/man/8/rbd.rst]: Looks like Nithya decided to leave it in.

**Review by idryomov — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/60269 - Squid backport
https://github.com/ceph/ceph/pull/60270 - Reef backport
https://github.com/ceph/ceph/pull/60271 - Quincy backport

**Pull request merged**